docs/tutorials/mastering.md

```diff
@@ -667,7 +627,7 @@ Log out of the VM.
 
 ## 3. Configure VMs for SD-Core Deployment
 
-This section covers setting up the SSH keys and installation of Juju and MicroK8s on VMs which are going to build up the infrastructure for SD-Core.
+This section covers setting up the SSH keys and installation of necessary tools on the VMs which are going to build up the infrastructure for SD-Core.
```

Comment:
It's not added with this PR but I think we do not need line 653 `Log out of the VM` there as we are already on the host node.

Reply:
True.
Removed.